src/main/java/com/powsybl/openloadflow/OpenLoadFlowParameters.java

```diff
@@ -115,49 +122,47 @@ public class OpenLoadFlowParameters extends AbstractExtension<LoadFlowParameters
-                                                                         TRANSFORMER_VOLTAGE_CONTROL_MODE_NAME,
-                                                                         DC_POWER_FACTOR_NAME,
-                                                                         MIN_PLAUSIBLE_TARGET_VOLTAGE_NAME,
-                                                                         MAX_PLAUSIBLE_TARGET_VOLTAGE_NAME,
-                                                                         MIN_REALISTIC_VOLTAGE_NAME,
-                                                                         MAX_REALISTIC_VOLTAGE_NAME,
-                                                                         REACTIVE_RANGE_CHECK_MODE_NAME,
-                                                                         LOW_IMPEDANCE_THRESHOLD_NAME,
-                                                                         NETWORK_CACHE_ENABLED_NAME,
-                                                                         SVC_VOLTAGE_MONITORING_NAME,
-                                                                         STATE_VECTOR_SCALING_MODE_NAME,
-                                                                         MAX_SLACK_BUS_COUNT_NAME);
+    public static final List<Parameter> SPECIFIC_PARAMETERS = List.of(
+        new Parameter(SLACK_BUS_SELECTION_MODE_PARAM_NAME, ParameterType.STRING, "Slack bus selection mode", SLACK_BUS_SELECTION_MODE_DEFAULT_VALUE.name(), Arrays.stream(SlackBusSelectionMode.values()).map(Enum::name).collect(Collectors.toList())),
+        new Parameter(SLACK_BUSES_IDS_PARAM_NAME, ParameterType.STRING, "Slack bus IDs", null),
+        new Parameter(LOW_IMPEDANCE_BRANCH_MODE_PARAM_NAME, ParameterType.STRING, "Low impedance branch mode", LOW_IMPEDANCE_BRANCH_MODE_DEFAULT_VALUE.name(), Arrays.stream(LowImpedanceBranchMode.values()).map(Enum::name).collect(Collectors.toList())),
+        new Parameter(VOLTAGE_REMOTE_CONTROL_PARAM_NAME, ParameterType.BOOLEAN, "Generator voltage remote control", VOLTAGE_REMOTE_CONTROL_DEFAULT_VALUE),
+        new Parameter(THROWS_EXCEPTION_IN_CASE_OF_SLACK_DISTRIBUTION_FAILURE_PARAM_NAME, ParameterType.BOOLEAN, "Throws an exception in case of slack distribution failure", THROWS_EXCEPTION_IN_CASE_OF_SLACK_DISTRIBUTION_FAILURE_DEFAULT_VALUE),
+        new Parameter(LOAD_POWER_FACTOR_CONSTANT_PARAM_NAME, ParameterType.BOOLEAN, "Load power factor is constant", LOAD_POWER_FACTOR_CONSTANT_DEFAULT_VALUE),
+        new Parameter(PLAUSIBLE_ACTIVE_POWER_LIMIT_PARAM_NAME, ParameterType.DOUBLE, "Plausible active power limit", LfNetworkParameters.PLAUSIBLE_ACTIVE_POWER_LIMIT_DEFAULT_VALUE),
+        new Parameter(SLACK_BUS_P_MAX_MISMATCH_NAME, ParameterType.DOUBLE, "Slack bus max active power mismatch", SLACK_BUS_P_MAX_MISMATCH_DEFAULT_VALUE),
+        new Parameter(VOLTAGE_PER_REACTIVE_POWER_CONTROL_NAME, ParameterType.BOOLEAN, "Reactive power / voltage slope", VOLTAGE_PER_REACTIVE_POWER_CONTROL_DEFAULT_VALUE),
```

Comment:
The slope is in kV by MVar. So maybe change for description `Voltage per reactive power slope`

Reply:
Done